infra/build/developer-tools-light/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Download and verify the integrity of the download first
+
+FROM sethvargo/hashicorp-installer:0.1.3 AS installer
+# Required to download and install Terraform
+ARG TERRAFORM_VERSION
+ENV TERRAFORM_VERSION ${TERRAFORM_VERSION}
+RUN /install-hashicorp-tool "terraform" "$TERRAFORM_VERSION"
+
+FROM alpine:3.12
```

Comment:
We probably should consider using one of the google's managed base image, e.g. ubuntu 18.04 (gcr.io/cloud-marketplace/google/ubuntu1804) as base for security purposes.

Reply:
ref: https://cloud.google.com/container-registry/docs/managed-base-images

Reply:
Good idea, though note that Ubuntu would likely increase the image size substantially.

Reply:
We can use ubuntu16.04. It's used by most Cloud Build's published base builder images (e.g. https://github.com/GoogleCloudPlatform/cloud-builders/blob/master/wget/Dockerfile#L1) and those images and its layers are cached in Cloud Build's VMs, so should be no time wasted for pulling the image even the base size is bigger than the alpine image.

Reply:
launcher.gcr.io/google/ubuntu16_04 or l.gcr.io/google/ubuntu16_04 was the old path for gcr.io/cloud-marketplace/google/ubuntu1604, we should use the cloud marketplace one instead.

Reply:
I just verified ubuntu1804 is also cached on Cloud Build VMs by default.

FETCHSOURCE
Fetching storage object: gs://xingao_cloudbuild/source/1615839015.59864-0a169fa6ce4e4e1a84ee18d02d97faf6.tgz#1615839015956184
Copying gs://xingao_cloudbuild/source/1615839015.59864-0a169fa6ce4e4e1a84ee18d02d97faf6.tgz#1615839015956184...
/ [1 files][  259.0 B/  259.0 B]                                                
Operation completed over 1 objects/259.0 B.                                      
BUILD
Pulling image: gcr.io/cloud-marketplace/google/ubuntu1804
Using default tag: latest
latest: Pulling from cloud-marketplace/google/ubuntu1804
9f2825a1003a: Already exists
7e79098fe9b4: Already exists
3c2cba919283: Already exists
Digest: sha256:d3d006e57a628ee5e03c5fbe6781baa8b8603c11463d44f1da06c92d75fcf343
Status: Downloaded newer image for gcr.io/cloud-marketplace/google/ubuntu1804:latest
gcr.io/cloud-marketplace/google/ubuntu1804:latest
test
PUSH
DONE

Reply:
@xingao267 I have switched the base image. Some tools `ca-certificates, bash` etc are preinstall and I added `wget`.

Reply:
Thanks!